Makefile

```diff
@@ -14,7 +14,7 @@
 
 PKG = sigs.k8s.io/azurelustre-csi-driver
 GIT_COMMIT ?= $(shell git rev-parse HEAD)
-REGISTRY ?= jusjin.azurecr.io
+REGISTRY ?= azurelustre.azurecr.io
```

Comment:
This will be MCR, right?

Reply:
This registry is only for development purpose